Makefile

```diff
@@ -1,13 +1,34 @@
+VENV_DIR = ./venv
```

Comment:
It is nicer to add a `@` before each shell command to not show the command itself. the output will be nicer.
i.e: `@docker-compose kill`

Reply:
thanks added by c49958389e68c2069c11355f006c6acbcfdd60bc. I also added a `clean` target to remove the virtual env whenever it's needed. Just in case.